nixos/doc/manual/release-notes/rl-1809.xml

```diff
@@ -241,6 +241,18 @@ $ nix-instantiate -E '(import &lt;nixpkgsunstable&gt; {}).gitFull'
      <literal>networking.networkmanager.dns</literal> instead.
     </para>
    </listitem>
+   <listitem>
+    <para>
+     The option <varname>services.kubernetes.apiserver.admissionControl</varname>
+     was renamed to <varname>services.kubernetes.apiserver.enableAdmissionPlugins</varname>.
+    </para>
+   </listitem>
+   <listitem>
+    <para>
+     Recommented way to access the Kubernetes Dashboard is with HTTPS (TLS)
```

Comment:
- t -> d (must have)
- with -> via (nice to have)

and perhaps specify a complete URL to access the kubernetes dashboard, if possible.(wish list)

Reply:
Will fix the typo in an upcoming PR. Thanks :-)